posts/integrating-civic-into-a-static-serverless-website-part-1-of-2.md

```diff
@@ -0,0 +1,231 @@
+   
+Finally, to get that domain to route to your distribution, we can add an alias to Route53. The easiest way to do this is to copy the domain name of the CloudFront distribution (not its CNAME), and setting it as the Alias Target in Route53. It should look something like this:
+
+![missing pic: Civic Alias](https://raw.githubusercontent.com/tylerjohnhaden/blog-usa/master/images/2019/02/civic_alias.png)
+
+p.s. This does not need to be a sub-domain. I just happen to be using the root domain `tylerjohnhaden.com` for something else.
+
+Once created, it will take a while to spin up, so let us continue with the rest of the stack. Uploading a new `index.html` or other resource to S3 takes a few seconds, so we can easily update our client side code later.
+
+# Setup your Civic integration
+
+In order to use the developer tools for Civic, we need to use our identity to create an account and Civic app. [Login](https://integrate.civic.com/login) to the their developer's portal. This will also give you an identity that can be used to test out the app later on. They have straight forward instructions on downloading the mobile app and verifying your identity.
+
+Checkout their [developer's guide](https://docs.civic.com) for an explanation of the service and APIs. They describe the flow using the following graphic.
+
+![missing pic: Civic Flow](https://raw.githubusercontent.com/tylerjohnhaden/blog-usa/master/images/2019/02/civic_flow.png)
+
+As you can see, they are pretty open about their data flow, which can give confidence to the application owner and users about what and how their data is being shared. Civic integration was designed as a security solution and is intended to be used with confidential and secure applications (so security through obscurity is not an option).
+
+Referencing the above diagram, we will need to implement the "User Agent" and "Application Server" which is where Civic's front and backend SDKs come in. But first, we have to set up our application using Civic's portal.
+
+### Create a new application
+
+![missing pic: Civic Enrollment](https://raw.githubusercontent.com/tylerjohnhaden/blog-usa/master/images/2019/02/civic_enrollment.png)
+
+This step should be straight forward, following their clean developer interface. All you need will be the name, domain, and a logo to display to users in their mobile app. The name and logo can be whatever you like, and we already went through the trouble of setting up our domain registration.
+
+The domain you whitelist here should be the one used for the static site. Only the client side code will need this origin set to initially retrieve the JWT token. Our API endpoint is not needed since the backend uses app secrets to verify it's identity. Whitelisting domains ensures Civic only sends the QR code to our website.
```

Comment:
```suggestion
The domain you whitelist here should be the one used for the static site. Only the client side code will need this origin set to initially retrieve the JWT token. Our API endpoint is not needed since the backend uses app secrets to verify its identity. Whitelisting domains ensures Civic only sends the QR code to our website.
```